production/helm/loki/Chart.yaml

```diff
@@ -4,7 +4,7 @@ name: loki
 description: Helm chart for Grafana Loki in simple, scalable mode
 type: application
 appVersion: 2.6.1
-version: 3.3.4
+version: 3.3.5
```

Comment:
I know this is a little annoying but main is at `3.4.0` now.

Reply:
Fixed. Also updated `production/helm/loki/CHANGELOG.md`